crates/gitbutler-branch-actions/src/remote.rs

```diff
@@ -116,34 +122,37 @@ pub fn branch_to_remote_branch(branch: &git2::Branch) -> Result<Option<RemoteBra
         }
     };
     let name = Refname::try_from(branch).context("could not get branch name");
-    match name {
-        Ok(name) => branch
-            .get()
-            .target()
-            .map(|sha| {
-                Ok(RemoteBranch {
-                    sha,
-                    upstream: if let Refname::Local(local_name) = &name {
-                        local_name.remote().cloned()
-                    } else {
-                        None
-                    },
-                    name,
-                    last_commit_timestamp_ms: commit
-                        .time()
-                        .seconds()
-                        .try_into()
-                        .map(|t: u128| t * 1000)
-                        .ok(),
-                    last_commit_author: commit
-                        .author()
-                        .name()
-                        .map(std::string::ToString::to_string),
-                })
-            })
-            .transpose(),
-        Err(_) => Ok(None),
-    }
+    let Ok(name) = name else { return Ok(None) };
```

Comment:
I've seen that sort of pattern elsewhere and I'm not sure if I like it. To me it looks quite cluttered having it all on one line.

If it is the preferred way of doing it though I guess I should get used it it

Reply:
The line isn't formatted, so I think it will look better after formatting.
But even then, if you don't like it, that should be enough to keep things as is.